sbt-plugin/src/main/scala/akka/grpc/sbt/AkkaGrpcPlugin.scala

```diff
@@ -38,13 +38,42 @@ object AkkaGrpcPlugin extends AutoPlugin {
       akkaGrpcCodeGenerators := GeneratorAndSettings(ScalaServerCodeGenerator, akkaGrpcCodeGeneratorSettings.value) :: Nil,
       akkaGrpcModelGenerators := Seq[Target]((JvmGenerator("scala", ScalaPbCodeGenerator), akkaGrpcCodeGeneratorSettings.value) -> sourceManaged.value),
 
+      PB.protoSources in Compile += target.value / "protobuf",
```

Comment:
I'll have to take a bit of time to digest and play with this monday - haven't worked that closely with those plugins yet. Which task puts files into `target/protobuf`?

Reply:
You are right, I think this is a copy/paste left-over we can get rid of.

Reply:
To answer your question: no task puts files in `target/protobuf`.